main/src/main/scala/sbt/internal/LogManager.scala

```diff
@@ -229,8 +229,13 @@ object LogManager {
   //     s
   //   }
```

Comment:
is this commented block relevant to this?

Reply:
I think so.It is useful when you use abstract logger type but now those are managed loggers.

Reply:
Not particularly to this bug. The commented out code provides fallback to the project scoping when global settings are NOT available.

```scala
   // if global logging levels are not explicitly set, set them from project settings
```